src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1701,7 +1701,7 @@ internal long SetRequestContent(HttpRequestMessage request, string content)
             if (content == null)
                 return 0;
 
-            Encoding encoding = null;
+            Encoding encoding = Encoding.UTF8;
```

Comment:
Is this right place for the change?
We have 
```c#
    internal static class ContentHelper
    {
        #region Constants

        // default codepage encoding for web content.  See RFC 2616.
        private const string _defaultCodePage = "ISO-8859-1";
```

Reply:
I checked and apparently most of the methods in the ContentHelper class are never used, so modifiying _defaultCodePage does not work. The ContentHelper class might need a rewrite, but I think it's beyond the scope of this small fix,

Reply:
The `encoding` is used below in `StreamHelper.EncodeToBytes()`, and if it is null the `encoding` defaults to `_defaultCodePage `.

Reply:
I fixed it, following your suggestion